### PR TITLE
Add nonstd-string-view-lite 1.3.0 release

### PR DESCRIPTION
**Pull request by mattyclarkson**

### CODE REVIEW

**Comment by mattyclarkson** [comment]
@sarum9in RFC

File: upstream.wrap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = string-view-lite-1.3.0
+
+source_url = https://codeload.github.com/martinmoene/string-view-lite/tar.gz/v1.3.0
+source_filename = string-view-lite-1.3.0.tar.gz
+source_hash = 405ae095bca3c63da28c72a3528369b9ba3996f1992f3ae90fcb01a9d8bdef38
```
Comment (sarum9in) [inline upstream.wrap]: Hash matches: ☒
 expected: 405ae095bca3c63da28c72a3528369b9ba3996f1992f3ae90fcb01a9d8bdef38
      got: d83adb0495ccfba50ae5a1af70bde04f33a26dc40599b0ce9e55f09c075daf23

Hi, could you please double check the hash value?

Comment (mattyclarkson) [reply upstream.wrap]: Not sure how that happened, have updated.

**Comment by mattyclarkson** [comment]
@sarum9in ready to go

**Pull request merged**